services/private_ip_manager/src/main/java/com/futurewei/alcor/privateipmanager/entity/IpAddrRange.java

```diff
@@ -41,6 +41,7 @@
     private long usedIps;
     private long totalIps;
     private IpAddrAllocator allocator;
+    private Map<String, IpAddrAlloc> ips;
```

Comment:
Just make sure IpAddrRange is also stored in Ignite, right?

Reply:
Yes, it store in ipAddrRangeCache.